build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -81,23 +81,23 @@
     <Rule Id="RS0027" Action="Error" />
   </Rules>
   <Rules AnalyzerId="System.Collections.Immutable.Analyzers" RuleNamespace="System.Collections.Immutable.Analyzers">
-    <Rule Id="RS0012" Action="Warning" />
+    <Rule Id="RS0012" Action="None" />
   </Rules>
   <Rules AnalyzerId="System.Runtime.Analyzers" RuleNamespace="System.Runtime.Analyzers">
     <Rule Id="CA1305" Action="None" />
     <Rule Id="CA1307" Action="None" />
     <Rule Id="CA1308" Action="None" />
     <Rule Id="CA1810" Action="None" />
     <Rule Id="CA1816" Action="None" />
-    <Rule Id="CA1825" Action="Warning" />
+    <Rule Id="CA1825" Action="None" />
     <Rule Id="CA2002" Action="None" />
     <Rule Id="CA2207" Action="None" />
     <Rule Id="CA2208" Action="None" />
     <Rule Id="CA2216" Action="None" />
     <Rule Id="CA2219" Action="None" />
     <Rule Id="CA2241" Action="None" />
     <Rule Id="CA2242" Action="None" />
-    <Rule Id="RS0014" Action="Warning" />
+    <Rule Id="RS0014" Action="None" />
```

Comment:
My understanding is because the walker would crash if the analyzer registered any operation actions, doesn't need to be using `IHasArgumentExpression` explicitly.